spec/active_remote/cached/argument_keys_spec.rb

```diff
@@ -2,24 +2,24 @@
 
 describe ::ActiveRemote::Cached::ArgumentKeys do
   it "does not mutate a string by default" do
-    ::ActiveRemote::Cached::ArgumentKeys.new("hello", {}).cache_key.must_equal("hello")
+    _(::ActiveRemote::Cached::ArgumentKeys.new("hello", {}).cache_key).must_equal("hello")
```

Comment:
These types of changes were made due to deprecation warnings from Minitest about calling methods like `must_equal` directly on objects.